crates/agglayer-config/src/lib.rs

```diff
@@ -131,6 +131,12 @@ pub struct Config {
 
     #[serde(default, skip_serializing_if = "crate::is_default")]
     pub grpc: GrpcConfig,
+
+    /// Extra Certificate signer per network.
+    #[serde(default)]
+    #[serde_as(as = "HashMap<DisplayFromStr, _>")]
+    #[serde(skip_serializing_if = "HashMap::is_empty")]
+    pub extra_certificate_signer: HashMap<u32, Address>,
```

Comment:
One more question, are we sure there would be only one extra certificate signer?